doc_user/UserManual.adoc

```diff
@@ -1886,8 +1887,8 @@ See <<add-a-therapy,Add a therapy>> function for a detailed description.
 
 To remove a therapy highlight one occurrence of the therapy (see box above) in the
 *_Therapy_* window and then press the **[.underline]##R##emove Therapy** button and the therapy is
-removed. Therapies can be removed before or after the final saving (see
-<<save-function-therapy,Save function>>).
+removed from the window and the *[.underline]##S##ave* button will be enabled (if not already enabled). 
+The removing of a therapy is not effective until it is confirmed with *[.underline]##S##ave* button (see <<save-function-therapy,Save function>>).
```

Comment:
it would be better that `confirmed with *[.underline]##S##ave* button` was `confirmed with the *[.underline]##S##ave* button`

Reply:
done, thanks!